src/client/initialize-app.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+import emptyFunction from '../utils/empty-function';
+
+function initializeShopifyPlatform(context, options, hull) {
+  const { customerId, accessToken } = options;
+
+  if (/^[0-9]+$/.test(customerId) && !accessToken) {
+    hull.api.put('services/shopify/customers/' + customerId).then(function() {
+      document.location.reload();
+    });
+  }
+
+  Hull.on('hull.user.logout', function() {
+    document.location = '/account/logout';
+  });
+}
+
+function getAppInitializer(app) {
+  if (app.type === 'platforms/shopify_shop') {
```

Comment:
Ok and ok

On Tue, May 12, 2015 at 9:07 AM, Romain Dardour <notifications@github.com>
wrote:

>> +function initializeShopifyPlatform(context, options, hull) {
>> +  const { customerId, accessToken } = options;
>> +
>> +  if (/^[0-9]+$/.test(customerId) && !accessToken) {
>> +    hull.api.put('services/shopify/customers/' + customerId).then(function() {
>> +      document.location.reload();
>> +    });
>> +  }
>> +
>> +  Hull.on('hull.user.logout', function() {
>> +    document.location = '/account/logout';
>> +  });
>> +}
>> +
>> +function getAppInitializer(app) {
>> +  if (app.type === 'platforms/shopify_shop') {
> no big deal, Thought this would be cleaner since we're retiring the 'app' concept from public visiblity
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hull/hull-js/pull/189/files#r30131017